src/WebTestAssertionsTrait.php

```diff
@@ -39,8 +39,9 @@ public static function assertSelectorExists(string $selector, string $message =
         $client = self::getClient();
 
         if ($client instanceof PantherClient) {
-            $element = self::findElement($selector);
-            self::assertNotNull($element, $message);
+            $by = $client::createWebDriverByFromLocator($selector);
```

Comment:
Why this change?

Reply:
`findElement()` can't return null, it throws an `NoSuchElementException` if no element found, so the `assertNotNull` always pass. 

Instead, I check if that the return value of `findElements` isn't an empty array.